src/daily-auths-report.tsx

```diff
@@ -173,7 +205,7 @@ function DailyAuthsReport(): VNode {
                   value: "date",
                   thresholds: utcDay,
                 },
-                fill: agency ? "friendly_name" : "agency",
+                fill: agency ? "friendly_name" : "steelblue",
                 title: (bin: ProcessedResult[]) => {
```

Comment:
Should we be returning a title anymore for the "All" agencies?

Reply:
Probably not! good catch

Reply:
Updated in d72a3828a891f5d514e7ac1036678ccf1a8ab01c